tests/tasks/generators/clusterloader/load.yaml

```diff
@@ -80,6 +83,7 @@ spec:
       aws eks $ENDPOINT_FLAG update-kubeconfig --name $(params.cluster-name) --region $(params.region)
```

Comment:
I guess you need to add `#!/bin/bash` at the beginning of the `script` field, since we don't want `set -e` which is the default.
Ref: https://tekton.dev/docs/pipelines/tasks/#running-scripts-within-steps